.travis.yml

```diff
@@ -1,16 +1,29 @@
 sudo: false
-git:
-  depth: 2
-branches:
-  except:
-    - /^v\d/
+dist: trusty
 language: node_js
-node_js:
-  - iojs
-  - node
-notifications:
-  email: false
+
+env:
+  matrix:
+    - NODE_VERSION="8" NPM_VERSION="5"
+    - NODE_VERSION="8" NPM_VERSION="6"
+    - NODE_VERSION="9" NPM_VERSION="5"
+    - NODE_VERSION="9" NPM_VERSION="6"
+    - NODE_VERSION="10" NPM_VERSION="5"
+    - NODE_VERSION="10" NPM_VERSION="6"
+
+before_install:
+  - pip install --user codecov
+  # Prepare node and npm
+  - nvm install $NODE_VERSION && nvm use $NODE_VERSION
+  - npm install -g npm@$NPM_VERSION
+  - npm install
+
+before_script:
+  - "sudo chown root /opt/google/chrome/chrome-sandbox"
+  - "sudo chmod 4755 /opt/google/chrome/chrome-sandbox"
+
 script:
+  - npm audit
```

Comment:
Not needed. Github automatically reports vulnerable dependencies. https://help.github.com/articles/about-security-alerts-for-vulnerable-dependencies/